app/scripts/environment/.gitignore

```
@@ -0,0 +1 @@
+environment_local*.js
```

Dejar una línea vacía al final
```suggestion
environment_local*.js

```